prime-router/src/main/resources/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml

```diff
@@ -588,6 +588,7 @@ elements:
     bundleProperty: '%resource.extension("https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2-date-time").value[x]'
     value: [ '%diagnosticReport.issued' ]
 
+  # TODO: ticket.  This transform causes a circular JSON structure when applied to a FHIR bundle generated from converting HL7
```

Comment:
I originally encountered this when trying to write tests for an HL7 message with a sender transform, but I worked around it by using the original-pipeline-transform (there were othe rreasons that it made impossible to use), but I didn't want to lose track of this and will open a bug before merging.